src/histogram.cc

```diff
@@ -183,15 +182,9 @@ void HistogramBase::Record(const FunctionCallbackInfo<Value>& args) {
   (*histogram)->Record(value);
 }
 
-void HistogramBase::FastRecord(Local<Value> unused,
-                               Local<Value> receiver,
-                               const int64_t value,
-                               FastApiCallbackOptions& options) {
-  if (value < 1) {
-    HandleScope scope(options.isolate);
-    THROW_ERR_OUT_OF_RANGE(options.isolate, "value is out of range");
-    return;
-  }
```

Comment:
`value` is validated in the JS layer, so an assertion check should be fine here.